### DISCUSSION TITLE
Create a public PPG member roster?

### DISCUSSION THREAD

**Discussion by joelnitta** [pteridogroup/ppg#77] [Ideas]
The mailing list, which is functionally a list of PPG members, is not currently public. As [mentioned](https://github.com/pteridogroup/ppg/issues/64#issuecomment-1932884376) by @crothfels, it would be helpful to have a list matching PPG member names to GitHub handles so we can tag people who may be relevant to a particular discussion.

The eventual PPG paper will include all member names, and assume that anybody participating wants to be included in that list. So I think it should be no problem to publish the names of participants ahead of the publication, but perhaps we could make additional information (GitHub handle, email address) opt-in?

**Comment by GuguYanez** [comment]
I agree with the proposal! :)
